package.json

```diff
@@ -21,13 +21,10 @@
     "main": "./out/extension.js",
     "contributes": {
         "commands": [
-            {
-                "command": "local-history.echo",
-                "title": "Local History"
-            },
             {
                 "command": "local-history.viewAllForActiveEditor",
-                "title": "Local History: Active Editor"
+                "title": "Local History: Active Editor",
+                "when": "editorTextFocus"
```

Comment:
I'm not sure why this context still works when no files have been opened, ideally it should only trigger if a file is currently active (and has focus).